src/context/FormContext.tsx

```diff
@@ -82,6 +93,13 @@ export const FormProvider = (props: { children: ReactNode }) => {
 
   const [userFlow, setUserFlow] = useState<Journey | null>(null);
 
+  const [statusHasChanged, setStatusHasChanged] = useState<boolean | null>(
+    null
+  );
+
+  const [statusChangeEvent, setStatusChangeEvent] =
```

Comment:
this could be controversial but I am handling this event separately. "developmentPlanEvent" prop is optional on it as it is set based on the user selection on the radio buttons page